Merge changes from pinger-bot

PerchunPak opened the pull request:
This merges changes from my project.

Small changelog on it, will be:
- Add support for translations. (plus make them optional)
- Remove `poetry.lock` from template folder.
  This should be generated **after** generating from template.
  Because I have broken syntax in `pyproject.toml` for supporting optional translations.
- Translate everything to English. Another languages only in `.po` files.
- Remove broken hook in pre-commit.
- Build documentation in `dirhtml` mode.

PerchunPak committed:
Merge changes from pinger-bot.

This merges changes from my project.

Small changelog on it, will be:
- Add support for translations. (plus make them optional)
- Remove `poetry.lock` from template folder.
  This should be generated *after* generating from template.
  Because I have broken syntax in `pyproject.toml` for supporting optional
  translations.
- Translate everything to English. Another languages only in `.po` files.
- Remove broken hook in pre-commit.
- Build documentation in `dirhtml` mode.

PerchunPak commented:
@cofob amount of changes here are **huge**, so I would like to listen your opinion on it.

cofob commented on .github/ISSUE_TEMPLATE/bug_report.yml:
```diff
@@ -50,26 +50,26 @@ body:
+      label: This isn't a duplicate
       options:
-        - label: В репозитории и в интернете нет дубликата этой ошибки.
+        - label: In repository and internet there is no duplicate of this error.
```
```
Internet.
^
```

PerchunPak replied:
Why?

cofob replied:
It is usually capitalized, as are proper nouns and names of states. [Wikipedia link](https://en.wikipedia.org/wiki/Internet)

PerchunPak replied:
Resolved in https://github.com/fire-square/fire-square-style/pull/51/commits/639fb974e92c8b9d628919c912be1bb0697d7f26.

cofob commented on .github/ISSUE_TEMPLATE/bug_report.yml:
```diff
@@ -1,42 +1,42 @@
     id: python_version
     attributes:
-      label: Версия Python
+      label: Python Version
```
```
Python version
       ^
```

PerchunPak replied:
Resolved in https://github.com/fire-square/fire-square-style/pull/51/commits/639fb974e92c8b9d628919c912be1bb0697d7f26.

cofob commented on .github/ISSUE_TEMPLATE/bug_report.yml:
```diff
@@ -1,42 +1,42 @@
     id: poetry_version
     attributes:
-      label: Версия poetry
+      label: poetry Version
```
```
Poetry version
^      ^
```

PerchunPak replied:
`poetry` is a name of tool, it shouldn't be capitalized. Same as `Django` shouldn't be just `django`.

cofob replied:
Ok, i understand this. But that the fuck version is capitalized?

PerchunPak replied:
Resolved in https://github.com/fire-square/fire-square-style/pull/51/commits/639fb974e92c8b9d628919c912be1bb0697d7f26.

cofob commented on CONTRIBUTING.md:
````diff
@@ -1,68 +1,70 @@
 ```
 
-Так же установим `pre commit hooks` в `git`:
+Also, let's install `pre commit hooks` in `git`:
````
"Also, let's install git pre-commit hooks:" maybe?

PerchunPak replied:
`git` and `pre-commit` two different tools, why they need to be squashed in one \` block?

cofob replied:
Maybe you confused an apostrophe with a formatting character? I have no mono formatting blocks in my text

PerchunPak replied:
There is ` characters in text

PerchunPak replied:
Oh, I understood. But why does it have to be stripped of formatting?

cofob replied:
Because it's useless in this case

cofob replied:
Resolved in [a31dfe7](https://github.com/fire-square/fire-square-style/pull/51/commits/a31dfe797341371fcb449a805e1b3ab87b1b4066)

PerchunPak replied:
Please do not commit without resolving conversation.

Why this is useless? It's fully following the style in this file.

cofob replied:
Code formatting must highlight commands or code. In that case that thing does highlight this?

PerchunPak replied:
Code formatting is three apostrophe blocks, not a single apostrophe. Single just for highlighting something, this not limited with code.

PerchunPak replied:
After long waiting cofob answered me in DM. Resolving this in https://github.com/fire-square/fire-square-style/pull/51/commits/639fb974e92c8b9d628919c912be1bb0697d7f26.

cofob commented on README.md:
```diff
@@ -5,44 +5,59 @@
 
-Пример стиля кода для всех Python проектов организации [Fire Square](https://github.com/fire-square).
-Пример можно использовать где угодно, теперь он не зависит от определенных проектов.
+Code template for all projects [Fire Square](https://github.com/fire-square) organization.
```
"firesquare", not "Fire Square"

PerchunPak replied:
Renamed in 43ef266d187d2accdf4b0e77c5911bec0f24d5b2.

cofob commented on {{ cookiecutter.project_name }}/.github/ISSUE_TEMPLATE/bug_report.yml:
```diff
@@ -1,42 +1,42 @@
     id: python_version
     attributes:
-      label: Версия Python
+      label: Python Version
```
Same as previous. Don't use capitals!

PerchunPak replied:
But why?

cofob replied:
Because it doesn't look good. Why the hell is Version capitalized? That's not according to the rules.

PerchunPak replied:
Resolved in https://github.com/fire-square/fire-square-style/pull/51/commits/639fb974e92c8b9d628919c912be1bb0697d7f26.

cofob reviewed (approved):
Great work! A couple of very minor things.

PerchunPak committed:
Rename `Fire Square` to `firesquare` in README.md

cofob committed:
Small changes in git-pre-commit-hooks section

PerchunPak committed:
Revert "Small changes in git-pre-commit-hooks section"

This reverts commit a31dfe797341371fcb449a805e1b3ab87b1b4066.

PerchunPak committed:
Resolve conversations

PerchunPak committed:
Merge branch 'master' into merge-changes-from-pingerbot

# Conflicts:
#	{{ cookiecutter.project_name }}/poetry.lock
#	{{ cookiecutter.project_name }}/pyproject.toml

PerchunPak committed:
Return correct TOML format to pyproject.toml in template

So dependabot will again update template's dependencies.

However, now user should handly remove `sphinx-intl` dependency, if they do
not want to use locales.
